queries.sql

```diff
@@ -6,3 +6,28 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- update the animals table by setting the species column to unspecified
+BEGIN TRANSACTION;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
```

Comment:
- [ ] According to the requirements, after making this change, kindly verify that the change was made before rollback and afterwards verify that the changes were undone e.g.

```
BEGIN; -- start transaction

UPDATE animals
SET species = 'unspecified'; -- make change

SELECT species from animals; -- verify that change was made
 ROLLBACK;

 SELECT species from animals; -- verify that change was undone
```

Reply:
@Ghiftee done with this change